things/service/src/main/java/org/eclipse/ditto/things/service/enforcement/ThingEnforcerActor.java

```diff
@@ -681,15 +681,15 @@ private CompletionStage<Void> performWotBasedThingMessageValidation(final Messag
         return resolveThingDefinition()
                 .thenComposeAsync(optThingDefinition -> {
                     if (messageDirection == MessageDirection.TO) {
-                        return thingModelValidator.validateThingActionInput(
+                        return DittoWotIntegration.get(context().system()).getWotThingModelValidator().validateThingActionInput(
```

Comment:
`DittoWotIntegration.get(context().system()).getWotThingModelValidator()` is duplicated 3 times - it could be assigned to one variable instead.